src/prisma/decorator.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+from prisma import Prisma
+from functools import wraps
+from datetime import timedelta
+from typing import Any, Callable
+
+
+__all__ = ('use_prisma',)
+
+CLIENT_NAME_DEFAULT_VALUE: str = 'db'
+
+
+def use_prisma(*args: Any, **kwargs: Any) -> Callable[..., Any]:
```

Comment:
Hey @LucaTabone – I'm just passing by here and have one thing to comment on – hope that's helpful!

Using `Any` and `...` in the type signature of a decorator obscures the types of the annotated function, which can be dangerous. You can use the generic [ParamSpec](https://docs.python.org/3/library/typing.html#typing.ParamSpec) to avoid this problem.

PS: will leave to @RobertCraigie whether this makes sense as a first class part of the library

Reply:
Hey @kkom :) Thanks a lot for your comment!

I agree! Unfortunately ParamSpec is available for Python 3.10+, but the tests within this repository test against 3.9 too.

Is there another way you see on how we could provide better type-hinting here in light of Python 3.9?

Reply:
Oh, I see! I don't have much experience creating Python libraries, just applications, but would https://pypi.org/project/typing-extensions/ help here?

We got the "@override" decorator in Python 3.11 in this way just now.